.lintr

```diff
@@ -2,7 +2,16 @@ linters: linters_with_tags(
     tags = NULL, # include all linters
     implicit_integer_linter = NULL,
     extraction_operator_linter = NULL,
-    undesirable_function_linter = NULL,
+    keyword_quote_linter = NULL,
+    undesirable_function_linter(
+      fun = c(
+         # Base messaging
+         "message" = "use cli::cli_inform()",
```

Comment:
really good idea. We could do with this in epinowcast

Reply:
Best thing I never knew I could do with `lintr`. It made the refactoring process very smooth. Another good idea for a blog post. Stole the idea from the [pkgdown `.lintr` file](https://github.com/r-lib/pkgdown/blob/819cede3fbbb2410912f4d49365177c9159b809f/.lintr.R), so all credits to the authors there.